tasks/eva_3205/dated_studies.tsv

```diff
@@ -0,0 +1,2605 @@
+project	taxid	date	study_type
+PRJEB2733	4896	2013-05	EVA study
+PRJEB3025	112509	2013-05	EVA study
+PRJEB3027	112509	2013-05	EVA study
+PRJEB3040	112509	2013-05	EVA study
+PRJEB3041	112509	2013-05	EVA study
+PRJEB3042	112509	2013-05	EVA study
+PRJEB4019	9606	2013-06	EVA study
+PRJEB4163	34506	2013-06	EVA study
+PRJEB4395	4081	2013-7	EVA study
```

Comment:
Out of curiosity, why are some of the months missing the leading zero?

Also it might be nice to include the code you used generate this tsv for ease of regenerating the charts - maybe just another cell in the notebook.

Reply:
Not sure why there are missing zeros. I've added them in.
The data comes from the SAB count spreadsheet and had been put together manually up until now.
Regenerating the EVA studies is simple and the query to ENA is included.
Regenerating the dbSNP studies involve querying the accessioning warehouse and I would not want to run that code again.
For the unmapped studies I queried the filesystem.
I added a comment to describe the origin of each studies